apiserver/facades/client/sshclient/facade.go

```diff
@@ -325,6 +341,60 @@ func (facade *Facade) getExecSecretToken(cloudSpec environscloudspec.CloudSpec,
 	return broker.GetSecretToken(k8sprovider.ExecRBACResourceName)
 }
 
+// PublicHostKeyForTarget returns the host key for the target entity. In addition, it also returns
+// the jump server's host key.
+func (facade *Facade) PublicHostKeyForTarget(arg params.SSHHostKeyRequestArg) params.SSHHostKeyResult {
```

Comment:
The `public` bit it making the name quite verbose now. I'm inclined now to say we should just call this `TargetHostKey` but I'm not fussed if you leave it as is.

Reply:
Let's just leave it, I'd rather be explicit like your initial comment stated than leave it a little ambiguous.